client/scripts/views/pages/notifications.ts

```diff
@@ -635,6 +635,156 @@ const AllCommunitiesNotifications: m.Component<{
   },
 };
 
+const Erc20ChainEventNotificationRow: m.Component<{
+  subscriptions: NotificationSubscription[];
+  chainInfo: ChainInfo;
+}, {
+  option: string;
+  loading: boolean;
+  expanded: boolean;
+}> = {
+  view: (vnode) => {
+    const { subscriptions, chainInfo } = vnode.attrs;
+    const subscriptionName = `${chainInfo.id}-transfer`;
+    const subscription = subscriptions.find((s) => {
+      return (
+        s.category === NotificationCategories.ChainEvent
+        && s.objectId === subscriptionName
+      );
+    });
+
+    if (subscription && subscription.immediateEmail) {
+      vnode.state.option = NOTIFICATION_ON_IMMEDIATE_EMAIL_OPTION;
+    } else if (subscription && subscription.isActive) {
+      vnode.state.option = NOTIFICATION_ON_OPTION;
+    } else {
+      vnode.state.option = NOTIFICATION_OFF_OPTION;
+    }
+
+    const ERC20_EVENT_LIST = ['transfer'];
```

Comment:
We also check in `notification_row` for event type `'approval'`, should that also be included here?